eng/ProdConPublish.csproj

```diff
@@ -24,29 +22,11 @@
 
   <Target Name="GetFilesToPublish">
     <ItemGroup>
-      <SymbolsPackagesToPublish Include="$(PackageOutputPath)%(PackageArtifact.Identity).*.symbols.nupkg" />
-      <PackagesToPublish
-        Include="$(PackageOutputPath)%(PackageArtifact.Identity).*.nupkg"
```

Comment:
This is what is causing the ProdCon builds to fail. The publish step was finding duplicate items with different manifest metadata because "`Microsoft.Extensions.DependencyInjection.*.nupkg`" and "`Microsoft.Extensions.DependencyInjection.Abstractions.*.nupkg`" overlap in what they match.